docs/reference/urllib3.request.rst

```diff
@@ -1,4 +1,9 @@
 urllib3.request()
 =================
 
+.. autoclass:: urllib3._request_methods.RequestMethods
```

Comment:
`RequestMethods` is a private API, where are we publically mentioning it? Inheritance chains?

Reply:
Yes,  Inheritance chains of class `PoolManager`